### DISCUSSION TITLE
Question about setting the Datadir variable

### DISCUSSION THREAD

**Discussion by drsig744** [nextcloud/all-in-one#4744] [Questions] [answered]
I run Nextcloud AIO through Portainer on Debian 12 and have an extra volume attached to `/mnt` that I intend for the Nextcloud data. I'm currently in testing phase so there is no concerns about data loss or anything.

I have the following variables set in the compose.yaml file. I assumed at lease `NEXTCLOUD_DATADIR` was required, but was not sure about `NEXTCLOUD_MOUNT`

`environment: `
`NEXTCLOUD_DATADIR: /mnt/test-data/nextcloud `
`NEXTCLOUD_MOUNT: /mnt/`

After the Nextcloud AIO setup has been completed I noticed that an `ncdata` folder was created under `/mnt`

But there is also a folder `/mnt/test-data/nextcloud`

Both folders appear to have the same folder structure and contain a new file `test.file` that I created under the admin user:

user@test_server:/mnt/ncdata# tree
├── admin
│   ├── cache
│   ├── files
│   │   ├── Documents
│   │   │   ├── **test.file**

user@test_server:/mnt/test-data/nextcloud# tree
├── admin
│   ├── cache
│   ├── files
│   │   ├── Documents
│   │   │   ├── **test.file**

Is this expected? Did I do something wrong?

**Comment by szaimen** [comment, marked as answer]
Hi, currently this is expected. However it will be adjusted with https://github.com/nextcloud/all-in-one/pull/4755

**Reply by drsig744** [reply]
Got it. Thanks!

**Reply by szaimen** [reply]
This is now released with v9.0.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
